### PR TITLE
chore(docs): update content-publisher

### PR DESCRIPTION
**Pull request by enddynayn**
Update Content-Publisher-Service Documentation structure
and content.

### CODE REVIEW

**Review by aramikm — APPROVED** [review]
Looks good to me!

**Review by saraswatpuneet — APPROVED** [review]
i like the improvements and reducing AI generated verbosity

File: developer-docs/content-publishing/README.md [hunk 1]
```diff
@@ -1,289 +1,172 @@
...
+   make setup-account
+   ```
 
-```sh
-docker compose up [-d] content-publishing-service-<service>
-```
+### Verification
 
-#### 3. Check the job in BullUI, to monitor job progress based on defined tests
+### Verification
```
Comment (mattheworris) [inline developer-docs/content-publishing/README.md:113]: Duplicated Heading
```suggestion
```

**Review by mattheworris — COMMENTED** [review]
- Read through changes
- One duplicated heading

**Pull request merged**